src/Model/GroundWaterTransport/gwt1mwt1.f90

```diff
@@ -102,9 +98,12 @@ subroutine mwt_create(packobj, id, ibcnum, inunit, iout, namemodel, pakname, &
     character(len=*), intent(in) :: namemodel
     character(len=*), intent(in) :: pakname
     type(TspFmiType), pointer :: fmi
+    real(DP), intent(in), pointer :: eqnsclfac !< governing equation scale factor
+    character(len=LENVARNAME), intent(in) :: dvt !< For GWT, set to "CONCENTRATION" in TspAptType
+    character(len=LENVARNAME), intent(in) :: dvu !< For GWT, set to "mass" in TspAptType
+    character(len=LENVARNAME), intent(in) :: dvua !< For GWT, set to "M" in TspAptType
```

Comment:
I think you should change all these character definitions to character(len=*), which means they will come in at whatever size they were passed in with.  That way if we ever change the size, we'd only need to do it in the calling module, instead of in here too.  I'd change it for the other advanced packages too.

Reply:
Good to know that's how that works.  Have changed in 7345130, changes applied to all advanced packages.